Collections/Sergey/Loger/MyLoger.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Collections.Sergey.Loger
+{
+    class MyLoger: ILoger
+    {
+        public List<string> Logs { get; private set; }
```

Comment:
this "private set" doesn't prevent list of logs to be modified. Just try to check